mongo/topic_store.go

```diff
@@ -155,9 +166,13 @@ func (m *Mongo) GetContent(id string, queryTypeFlags int) (*models.ContentRespon
 		contentSelect["current.timeseries"] = 1
 	}
 
-	err := s.DB(m.Database).C(m.ContentCollection).Find(bson.M{"id": id}).Select(contentSelect).One(&content)
+	err := m.Connection.
```

Comment:
The topic api uses two collections. At this line it is clear that the ContentCollection is being used.
Please change lines: 90, 104 to follow same pattern and adjust other code elsewhere to accomodate such as line 34 which seems to be missing the content collection.

Reply:
@redhug1 The default collection is being configured in the mongo Connection creation to facilitate the use case of where the same code snippet of checking a collection is used in all the database methods. 
`GetConfiguredCollection()` is supposed to be used to remove this redudant duplication.

A service using multiple collections is an exception rather than the norm. 

However for the edge cases where this is needed as in this use case, we are also having a way to select the other collection.

Reply:
@redhug1 The default collection is being configured in the mongo Connection creation to facilitate the use case of where the same code snippet of checking a collection is used in all the database methods. 
`GetConfiguredCollection()` is supposed to be used to remove this redudant duplication.

A service using multiple collections is an exception rather than the norm. 

However for the edge cases where this is needed as in this use case, we are also having a way to select the other collection which is using `C` method similar to the mgo driver API.